lib/streamlit/source_util.py

```diff
@@ -157,12 +163,16 @@ def get_pages(main_script_path_str: str) -> Dict[str, Dict[str, str]]:
         )
 
         for script_path in page_scripts:
+            script_path_str = str(script_path)
             pn, pi = page_name_and_icon(script_path)
-            if pn in used_page_names:
-                continue
+            psh = calc_md5(script_path_str)
```

Comment:
what is psh? page script hash? I feel like page_script_hash is a better variable name but not sure if it conflicts with anything.

Reply:
This is kinda style dependent. Some languages idiomatically prefer short variable names if the scope of the variable is small enough that it's unlikely you can forget what the variable is (and the loop body here is <10 lines)